.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+	"env": {
+		"browser": true,
+		"es2021": true,
+		"node": true
+	},
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:jest/recommended",
+		"plugin:prettier/recommended",
+		"prettier/@typescript-eslint",
+		"prettier/react"
+	],
+	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
```

Comment:
Why not use `project` property instead and let eslint parse the `tsconfig.json`?